lib/web_console/middleware.rb

```diff
@@ -19,12 +14,15 @@ class Middleware
 
     UNACCEPTABLE_REQUEST_MESSAGE = "A supported version is expected in the Accept header."
 
+    cattr_accessor :mount_point
+    @@mount_point = '/__web_console'
+
     cattr_accessor :whiny_requests
     @@whiny_requests = true
 
     def initialize(app, options = {})
```

Comment:
Actually, why not drop the options at all? We'll save a couple of allocations and we can still configure the middleware through rails.